# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: intl/hyphenation/glue/nsHyphenationManager.cpp
# Commit: 6450088b6b73
# Full Hash: 6450088b6b73ffc17c79cd7097b1bfe30d00e207
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2020-09-14 21:29:41
# Regressor Bug: 1627075
# File Overlap Count: 1
# Description:
#   Bug 1656261 - Back out all recent StartupCache work r=RyanVM
#   
#   This backs out all work from bug 1627075 as well as all of its
#   descendents. There were a few conflicts when backing this out but
#   overall it was pretty clean, so I would say it's a fairly mild
# ==============================================================================

diff -r b0fabcd60430 -r 6450088b6b73 intl/hyphenation/glue/nsHyphenationManager.cpp
--- a/intl/hyphenation/glue/nsHyphenationManager.cpp	Mon Sep 14 17:32:40 2020 +0000
+++ b/intl/hyphenation/glue/nsHyphenationManager.cpp	Mon Sep 14 17:00:53 2020 +0000
@@ -227,7 +227,7 @@
     return;
   }
 
-  RefPtr<CacheAwareZipReader> zip = Omnijar::GetReader(aType);
+  RefPtr<nsZipArchive> zip = Omnijar::GetReader(aType);
   if (!zip) {
     return;
   }